frontend/tests/playwright/product/product_image.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { test, expect, Page } from '@playwright/test';
+
+test.describe('Product image test', () => {
+   test('product with a single image ', async ({ page }) => {
+      await page.goto('/products/iflex-opening-tool');
+
+      const image = page
+         .getByRole('img', { name: 'iFlex Opening Tool' })
+         .first();
+      expect(image).toBeVisible;
+   });
+
+   test('product with multiple images ', async ({ page }) => {
+      await page.goto('/products/repair-business-toolkit');
+
+      const firstImageSrc = await getCurrentActiveImgSrc(page);
+      const firstImage = page.locator(`img[src="${firstImageSrc}"]`).first();
+      await expect(firstImage).toBeVisible();
+
+      // Click on the arrow for the next image
+      await page.getByTestId('swiper-next-image').first().click();
```

Comment:
Do we need `first()`? I thought it automatically got the first one if another action is appended to it.

```suggestion
      await page.getByTestId('swiper-next-image').click();
```

Reply:
I had to use `.first()` because it was throwing an error because it found 2 elements:

<img width="887" alt="Screen Shot 2023-01-05 at 6 06 57 PM" src="https://user-images.githubusercontent.com/58952979/210915890-cc892422-c9c0-40b6-b245-ed2a9febb173.png">

> I thought it automatically got the first one if another action is appended to it.

This, I cannot confirm but it seems like it fails when it finds multiple elements.

Reply:
TIL!

Reply:
To note, playwright will throw an error if more than one matching element is found https://playwright.dev/docs/locators#throws-an-error-if-more-than-one